packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -13,7 +13,7 @@
 	.components-base-control,
 	.components-radio-control {
 		&:where(:not(:last-child)) {
-			margin-bottom: $grid-unit-20;
+			margin-bottom: $grid-unit-20 !important;
```

Comment:
I'm not sure about this change, as it feels like it could have an unexpected knock-on effect with other UI controls. The classnames it affects are very broadly used.

I think adding some specific styles to the `DateFormatPicker` component's stylesheet would help limit the scope of the change.

You might need to add a classname to the ToggleControl to be able to style it.